.claude/agents/pr-agent.md

```
@@ -93,8 +168,10 @@ PR 생성 완료 후:
 - `--no-verify`, `--force` 사용 금지
 - main/develop에 직접 커밋/푸시 금지
 - PR 설명 없이 생성 금지
-- 테스트 미통과 상태에서 PR 생성 금지 (CI가 잡겠지만, 올리기 전에 확인)
+- 테스트 미통과 상태에서 PR 생성 금지
+- **리뷰 게이트(5단계) CRITICAL 미해결 상태에서 PR 생성 절대 금지**
 
 ## 사용자에게 확인받아야 하는 것
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)